# Used by BuilderPlugin

uint8 ADD=1
uint8 UPDATE_POSE=2
uint8 DELETE=3

uint8 action
string id
string type
geometry_msgs/Pose pose
---
string error_msg
